widgetssdk/src/main/java/com/glia/widgets/chat/ChatManager.kt

```diff
@@ -128,7 +128,7 @@ internal class ChatManager constructor(
 
     @VisibleForTesting
     fun checkUnsentMessages(state: State) {
-        sendUnsentMessagesUseCase(state.unsentItems.firstOrNull()?.message ?: return) {}
+        sendUnsentMessagesUseCase(state.unsentItems.firstOrNull() ?: return) {}
     }
```

Comment:
The name of the functions makes me think that it's possible that several messages can be unsent.

However, as far as I understand, `state.unsentItems.firstOrNull()` will pass only one message to `sendUnsentMessagesUseCase`.

So, my concern is: won't we loose other unsent messages?

Reply:
When a response is successfully sent, the message is received through a socket and removed from the unsent messages list, and the method is called again until the list is empty. This is a kind of recursive method based on socket callback.